js/create_project_progress.jsx

```diff
@@ -0,0 +1,83 @@
+var DeployKeyTest = require('./deploy_key_test.jsx');
+
+var CreateProjectProgress = React.createClass({
+
+	checkInterval: false,
+
+	getInitialState: function() {
+		return {
+			complete: false,
+			checkingStatus: false
+		};
+	},
+
+	componentDidMount: function() {
+		this.checkInterval = window.setInterval(this.handleCheckStatus, 10000);
```

Comment:
We should probably get into a habit of cleaning stuff up - how about you kill off the timer in componentWillUnmount?